### PR TITLE
adding payments line

### PR DESCRIPTION
**Pull request by laurenreidexpensify**
Adding an extra line of detail to direct any payment queries to the contributors@ email.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ If you are hired for an Upwork job and have any job-specific questions, please a
...
+We hire and pay external contributors via Upwork.com. If you'd like to be paid for contributing, please create an Upwork account and apply for a job in the [Upwork issue list](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2). Payment for your contributions will be made no less than 7 days after the pull request is merged to allow for regression testing. We hire one contributor for each Upwork job. New Expensify.cash contributors are limited to working on one job at a time, however experienced contributors may work on numerous jobs simultaneously. If you have not received payment after 8 days, please email contributors@expensify.com referencing the GH issue and your GH handle.
```
Comment (iwiznia) [inline CONTRIBUTING.md:22]: I think you should clarify: `after 8 days of the PR being deployed to production`

Comment (iwiznia) [inline CONTRIBUTING.md:22]: I think you should clarify: `after 8 days of the PR being deployed to production`

**Review by iwiznia — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (laurenreidexpensify) [reply CONTRIBUTING.md]: updated here https://github.com/Expensify/Expensify.cash/pull/2407/commits/dfea1c64e01f6145ad34e8ef1f55b4f5e85e784b

**Review by joelbettner — APPROVED** [review]

**Review by iwiznia — APPROVED** [review]

**Comment by michaelhaxhiu** [comment]
Nice one, Lauren :)

**Review by michaelhaxhiu — COMMENTED** [review]
thumbs up from me

**Comment by michaelhaxhiu** [comment]
Why won't this PR let me change this to a ✅ ?

image

Is it because it was merged before I actioned the PR review? I played with this for like 30 minutes yesterday, and thought that leaving a comment would do the trick but it didn't. Just wanna make sure I understand.

**Comment by iwiznia** [comment]
> Is it because it was merged before I actioned the PR review? 

Yes

**Pull request merged**